docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/is-latest-version-of-docker-sdk-installed.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Is the latest version of Docker SDK installed?
+description: This document allows you to assess if the latest version of Docker SDK is installed in a project.
+template: howto-guide-template
+---
+
+# Is latest version of Docker SDK installed?
+
+{% info_block infoBox %}
+
+Resources: Backend
+
+{% endinfo_block %}
+
+## Description
+
+### With project repository access
+
+1. Find `docker` folder in the root of the project and ensure it’s sub-repository.
+2. Run `git remote show origin` from the `docker` folder in the root of the project and ensure that the origin is pointing to `git@github.com:spryker/docker-sdk.git`.
+3. Run `git log` from `docker` submodule. Having that information it’s easier to judge how much docker sdk is outdated comparing it with [releases](https://github.com/spryker/docker-sdk/releases)
```

Comment:
1. How is the `docker` submodule different from the `docker` folder?
2. What exactly should I be comparing to the releases?

Reply:
1. in this case, I think it would be better to use `submodule` for both points. it should be fixed for better consistency.
2. the latest commit from the docker submodule should be compared with our Docker SDK repository to check how outdated the project version is.